app/src/components/integrations/elements/integrationSettings/integrationSettings.jsx

```diff
@@ -74,8 +87,7 @@ export const IntegrationSettings = (props) => {
   );
 
   const testIntegrationConnection = useCallback(() => {
-    setLoading(true);
-    if ('id' in props.data) {
+    if ('id' in props.data && pluginName !== AD && pluginName !== LDAP && pluginName !== SAML) {
```

Comment:
As for me, such a long condition looks bad.

May be in is better f.e. create an object such as authPlugins={
[AD]:AD,
...
}
and use it like this 
if('id' in data && !authPlugins[pluginName])

Reply:
Reverted all changes